examples/combobox/js/combobox-autocomplete.js

```diff
@@ -254,7 +254,7 @@ ComboboxAutocomplete.prototype.getNextOption = function (currentOption) {
 /* MENU DISPLAY METHODS */
 
 ComboboxAutocomplete.prototype.doesOptionHaveFocus = function () {
-  return this.combobocNode.getAttribute('aria-activedescendant') !== '';
+  return this.comboboxNode.getAttribute('aria-activedescendant') !== '';
```

Comment:
This appears to actually be  bug

Reply:
@nschonni 
Yes it is, so you can fix it.
I need to look at the combobox code again, I think it needs to be updated, there are better was to do what this code is trying to accomplish.